rust/processor/src/config.rs

```diff
@@ -65,6 +69,11 @@ impl IndexerGrpcProcessorConfig {
     pub const fn default_pb_channel_txn_chunk_size() -> usize {
         100_000
     }
+
+    /// Default timeout for grpc response item in milliseconds. Defaults to 300 seconds.
+    pub const fn default_grpc_response_item_timeout_ms() -> u64 {
```

Comment:
could we make this return a duration instead of seconds? this is in line with other config values [https://github.com/aptos-labs/aptos-indexer-processors/blob/41461d455dbeb1699159ab1dd91f30b9b074c5db/rust/processor/src/config.rs#L122](link)

Reply:
done done done. sorry for the delay 🤦